rpm/containerd.spec

```diff
@@ -30,8 +30,8 @@ AutoReq: no
 
 Name: containerd.io
 Provides: containerd
-# For some reason on rhel 8 if we "provide" runc then it makes this package unsearchable
-%if 0%{!?el8:1}
+# For some reason on rhel >= 8 if we "provide" runc then it makes this package unsearchable
+%if 0%{?rhel} <= 7
```

Comment:
This may also be excluding this rule on Fedora, (open)SUSE, Oracle etc (assuming those don't have `rhel` defined), so a larger change than only "exclude on RHEL >= 8